app/Controller/PagesController.php

```diff
@@ -74,3 +120,44 @@ public function display() {
 		}
 	}
 }
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
+    public function example()  {
```

Comment:
[phpmd] The method example() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10.